docs/src/model_setup/grids.md

```diff
@@ -51,6 +52,15 @@ RegularCartesianGrid{Float64, Periodic, Bounded, Bounded}
 grid spacing (Δx, Δy, Δz): (156.25, 156.25, 31.25)
 ```
 
+Note that currently only a subset of the possible topologies are available (the limiting factor being the pressure solver):
+
+```
+Topology = (Periodic, Periodic, Periodic)
+Topology = (Periodic, Periodic, Bounded)
+Topology = (Periodic, Bounded, Bounded)
+Topology = (Bounded, Bounded, Bounded) # Note: only works on CPUs
```

Comment:
We prefer to use lowercase (snake-case) for variables, and reserve capitalized names for `struct`s and `module`s (roughly following https://github.com/jrevels/YASGuide). So these should all be `topology = ...`.

Reply:
(and the comment is nice :-D)